paasta_tools/cli/cmds/status.py

```diff
@@ -1715,7 +1716,7 @@ def get_replica_state(pod: KubernetesPodV2) -> ReplicaState:
             state = ReplicaState.UNKNOWN
 
         # Catch
-    elif phase == "Failed":
+    elif phase == "Failed" or reason == "Evicted":
         # e.g. pod.reason == evicted
```

Comment:
might be worth cleaning up these comments too - the current formatting is a bit confusing (and after adding this new conditional, i don't think they really add anything)